test/express.test.js

```diff
@@ -1,27 +1,26 @@
+const fs = require('fs');
+const path = require('path');
+
 const fetch = require('node-fetch');
+const FormData = require('form-data');
 
 const Server = require('../servers/server');
 
 const { expect } = require('chai');
 
 describe('Server tests', function () {
-  before(async function() {
-    testServer = new Server(['./servers/express.js','http:127.0.0.1:8080','--logfile=./test/sample-data.log']);
-    return testServer.readyPromise;
-  });
-
-  after(async function() {
+  let testServer;
+  afterEach(async function() {
     return testServer.stop({type: 'signal', value: 'SIGKILL'});
   });
 
-  it('returns correct data', async function () {
-    const url = 'http:127.0.0.1:8080/api?first=1655888897602&last=1655888906678&timeseries=eventloop&timeseries=memory';
-    const response = await fetch(url);
-    const data = await response.json();
+  it('returns correct data when provided a static file', async function() {
```

Comment:
maybe it would make sense to separate this test (the only one that has the `--logfile` argument) from the others. then there can a single `beforeEach()` that starts the server and that can be refactored out of the other tests.